frontend/src/components/pages/RegistrantExperience/index.tsx

```diff
@@ -125,8 +130,10 @@ const RegistrantExperiencePage = (): React.ReactElement => {
   };
 
   useEffect(() => {
-    if (!isLoading.camp && !isLoading.waiver) setCurrentStep("selection");
-  }, [isLoading]);
+    if (!isLoading.camp && !isLoading.waiver) {
+      setCurrentStep(camp && waiver ? "selection" : "notfound");
```

Comment:
This is what we did in the previous version of this file (with the `camp && waiver`).

Reply:
Is there current support for the notfound step? I see that we have the Notfound component but it doesn't seem to be used

Reply:
Yeah there is, the default case will be invoked when we set `currentStep` to `notFound`. Without this, we would never reach the default case which resulted in the error. I've added a little comment to make this clear beside the default case in `getCurrentStep`.